auto_cpufreq/core.py

```diff
@@ -200,60 +197,14 @@ def charging():
     """
     get charge state: is battery charging or discharging
     """
-    power_supply_path = "/sys/class/power_supply/"
-    power_supplies = os.listdir(Path(power_supply_path))
-    # sort it so AC is 'always' first
-    power_supplies = sorted(power_supplies)
-
-    # check if we found power supplies. on a desktop these are not found
-    # and we assume we are on a powercable.
-    if len(power_supplies) == 0:
-        # nothing found found, so nothing to check
-        return True
-    # we found some power supplies, lets check their state
-    else:
-        for supply in power_supplies:
-            # Check if supply is in ignore list
-            ignore_supply = any(item in supply for item in POWER_SUPPLY_IGNORELIST)
-            # If found in ignore list, skip it.
-            if ignore_supply:
-                continue
+    return True if psutil.sensors_battery().power_plugged else False
 
```

Comment:
Using psutil to get charging status

Reply:
As I mentioned in other comment this was all added with #308 as battery state would not be showing properly on certain devices, still thinking if we might need some of it or if `psutil` will be able to pick it all up itself ...